modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -151,3 +151,11 @@ endif::[]
 
 {oracle} clients are registered in the same way as all other clients.
 For more information, see xref:client-configuration:registration-overview.adoc[].
+
+
+[WARNING]
```

Comment:
Same as for CentOS. Only Oracle6 has this problem. Rest of my comments for CentOS6 also apply here.